server_ftp/data/server_ftp_data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Make all this into demo. Actually not all of this. One ftp server with two folders is quite enough. Put in a realisticly looking url like ftp://ftp.example.com

Reply:
Actually we need to change url into host, as the argument is a hostname and not an url